frontend/playwright.config.ts

```diff
@@ -13,13 +13,13 @@ import { devices } from '@playwright/test';
 const config: PlaywrightTestConfig = {
    testDir: './tests/playwright',
    /* Maximum time one test can run for. */
-   timeout: 30 * 1000,
+   timeout: 60 * 1000,
    expect: {
       /**
        * Maximum time expect() should wait for the condition to be met.
        * For example in `await expect(locator).toHaveText();`
        */
-      timeout: 5000,
+      timeout: 10000,
```

Comment:
Cool! On our phpunit WebDriver tests in ifixit/ifixit we wait for 30 seconds. We may need to expand this in the future.

Reply:
Is the `30 seconds` for the assertion or for the test case?

Reply:
Assertion. Basically, if we call `waitForPageLoadAfter` (or any of the wait calls) we default to 30 seconds. It may be excessive for these tests.